### ISSUE TITLE
image reconstruction result comparision of the pretrained ConvNeXt-V1/V2 model

### ISSUE THREAD

**Issue by songkq** [facebookresearch/ConvNeXt-V2#11]
@shwoo93 @s9xie  hi, did you make a comparision about the effect of image reconstruction of the pretrained V1/V2 model? How about comparing with MAE-pretrained ViT?

**Comment by s9xie** [comment]
Yes we have visualized the reconstruction results several times and found the reconstructed results very similar to those obtained by ViT MAE (or Swin SimMIM), even before adding the GRN layer (ie. using the v1 model). In other words, it seems one cannot really judge the representation quality from the pixel space reconstruction quality: even we get “perfect” reconstructions, the finetuning results can still have a large gap.

**Comment by songkq** [comment]
@s9xie Does ConvNeXt-v1/v2 model equipped without sparse conv also work well in image reconstruction?

**Comment by songkq** [comment]
> Yes we have visualized the reconstruction results several times and found the reconstructed results very similar to those obtained by ViT MAE (or Swin SimMIM), even before adding the GRN layer (ie. using the v1 model). In other words, it seems one cannot really judge the representation quality from the pixel space reconstruction quality: even we get “perfect” reconstructions, the finetuning results can still have a large gap.

Since the gap between pretraining and finetuning, could we introcude the MAE-based ConvNeXt Encoder in a Semi-supvised framework such as FixMatch?

**Comment by songkq** [comment]
@s9xie @shwoo93 Did you make a comparison among the supervised ConvNeXt-v2 models across different model sizes?
![image](https://user-images.githubusercontent.com/30183023/211258643-e9b7abd3-f7a9-4ae4-a602-275f1cbaa0ab.png)
